### PR TITLE
Audio Colours

### PR DESCRIPTION
**Pull request by oliverlloyd**
The little dot on the range slider was not picking up the pillar colour so this adds the styles to do that

### Before
Screenshot 2020-11-09 at 13 02 22

### After
Screenshot 2020-11-09 at 13 01 31

### CODE REVIEW

**Comment by oliverlloyd** [comment]
[Always blue](https://trello.com/c/x1esBaLc/2135-always-blue)

File: src/AudioAtom.tsx [hunk 1]
```diff
@@ -116,6 +116,28 @@ const progressBarInputStyle = (pillar: Pillar) => css`
...
+        border: none;
+        border-radius: 50px;
+    }
+    &::-ms-thumb {
+        background: ${pillarPalette[pillar][400]};
+        width: 14px;
+        height: 14px;
+        border: none;
+        border-radius: 50px;
+    }
```
Comment (liywjl) [inline src/AudioAtom.tsx:140]:
```suggestion
    &::-webkit-slider-thumb, &::-moz-range-thumb, &::-ms-thumb {
        background: ${pillarPalette[pillar][400]};
        -webkit-appearance: none;
        width: 14px;
        height: 14px;
        border-radius: 50px;
    }
```

**Review by liywjl — APPROVED** [review]
Great catch! The thumb CSS selector is non standard, but that is not something we can control.

File: src/AudioAtom.tsx (hunk 1 above)
Comment (oliverlloyd) [reply src/AudioAtom.tsx:140]: In [this css-tricks article](https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/) it says you can't group these selectors like that:

> Note that while we’re repeating code here, that’s necessary as you can’t comma-separate these type of selectors. Browsers will drop the entire selector if it doesn’t understand a part of it.

I haven't tried this though

Comment (liywjl) [reply src/AudioAtom.tsx:140]: cool! yes then you can ignore

**Pull request merged**